end2end-test-examples/gcs/src/main/java/io/grpc/gcs/GcsioClient.java

```diff
@@ -32,8 +32,9 @@ public class GcsioClient {
   private ObjectResolver objectResolver;
   private GoogleCloudStorageOptions gcsOpts;
   private GoogleCredential creds;
+  private int gRPCVersion;
 
-  public GcsioClient(Args args, boolean grpcEnabled) throws IOException {
+  public GcsioClient(Args args, int gRPCVersion) throws IOException {
```

Comment:
What about keeping` grpcEnable` flag and passing in the new `clientType`? That was we can still use this test  for various combination of `gcsio`

1. gRPC via oldClient
1. gRPC via java-storage
2. Json  using Apiary client 
3. Json using java-storage client (will be there in future)

Reply:
I do understand we will not have gcsio's `ClientType` at caller of these tests but I guess we can hardcode the client type there. It's not something which we expect to modify frequently.